ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -140,6 +140,15 @@ PMP_SS_RMM:
   - low_pressure_status
   - high_pressure_status
 
+PMP_SS_RMM_CPC:
+  description: "Pump with circulation pump and run mode monitoring."
+  is_canonical: true
+  implements:
+  - PMP
+  - SS
+  - RMM
+  - CPC
```

Comment:
`CPC` should be used on non-PMP types. For example, if a boiler had a pump that was internal to the unit. Can you please clarify; are you trying to model a stand-alone pump or a circulation pump that is inside of another piece of equipment?